packages/composite-checkout-wpcom/src/components/form-field-annotation.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import styled from '@emotion/styled';
+
+type FormFieldAnnotationProps = {
+	// Semantic props
+	labelText: string;
+	descriptionText?: string;
+	errorMessage: string;
+
+	// Functional props
+	isError: boolean;
+	isDisabled: boolean;
+
+	// Technical props
+	className?: string;
+	labelId: string;
+	descriptionId: string;
+	formFieldId: string;
+};
+
+export const FormFieldAnnotation: FunctionComponent< FormFieldAnnotationProps > = ( {
+	formFieldId,
+	labelText,
+	labelId,
+	descriptionText,
+	descriptionId,
+	errorMessage,
+	isError,
+	isDisabled,
+	className,
+	children,
+} ) => (
+	<div className={ className }>
+		{ labelText && (
```

Comment:
The `labelText` prop is declared as mandatory in `FormFieldAnnotationProps`, but the code suggests it's optional.

Reply:
Good catch- I changed the component body to make the label required.